frontend/test/fixture/user/globalManagerSetting.js

```diff
@@ -0,0 +1,11 @@
+import { ROLE } from 'tracim_frontend_lib'
+
+export const globalManagerSetting = {
+  role: ROLE.workspaceManager.slug,
+  email_notification_type: 'summary'
+}
+
+export const globalManagerSettingUpdated = {
```

Comment:
what means "Updated" ?

Reply:
it's new setting for global manager to test the update of setting